editor/editor_inspector.cpp

```diff
@@ -3365,7 +3365,10 @@ void EditorInspector::set_keying(bool p_active) {
 		return;
 	}
 	keying = p_active;
-	update_tree();
+	// Propagate the keying state to its editor properties.
+	Array args;
+	args.append(keying);
+	main_vbox->propagate_call(SNAME("set_keying"), args, true);
```

Comment:
I think you can do
```suggestion
	main_vbox->propagate_call(SNAME("set_keying"), { keying }, true);
```

Reply:
It doesn't seem to work, and an error is reported when compiling: `error: no matching constructor for initialization of 'const Array &'`.

Reply:
Ah right, it's only possible with Vector.